File: 02-01-2026/equation_validation.py
def is_valid_equation(equation):
    left_side, right_side = equation.split('=')
    left_value = eval(left_side.strip())    


    return left_value == float(right_side.strip())

print(is_valid_equation("2 + 2 = 4")); # true.
print(is_valid_equation("2 + 3 - 1 = 4")); # true.
print(is_valid_equation("8 / 2 = 4")); # true.
print(is_valid_equation("10 * 5 = 50")); # true.
print(is_valid_equation("2 - 2 = 0")); # true.
print(is_valid_equation("2 + 9 / 3 = 5")); # true.
print(is_valid_equation("20 - 2 * 3 = 14")); # true.
print(is_valid_equation("2 + 5 = 6")); # false.
print(is_valid_equation("10 - 2 * 3 = 24")); # false.
print(is_valid_equation("3 + 9 / 3 = 4")); # false.

File: 02-01-2026/equation_validation.js
function isValidEquation(equation) {
  const [left, right] = equation.split("=");
  const leftValue = eval(left.trim());

  return leftValue === parseFloat(right.trim());
}

console.log(isValidEquation("2 + 2 = 4")); // true.
console.log(isValidEquation("2 + 3 - 1 = 4")); // true.
console.log(isValidEquation("8 / 2 = 4")); // true.
console.log(isValidEquation("10 * 5 = 50")); // true.
console.log(isValidEquation("2 - 2 = 0")); // true.
console.log(isValidEquation("2 + 9 / 3 = 5")); // true.
console.log(isValidEquation("20 - 2 * 3 = 14")); // true.
console.log(isValidEquation("2 + 5 = 6")); // false.
console.log(isValidEquation("10 - 2 * 3 = 24")); // false.
console.log(isValidEquation("3 + 9 / 3 = 4")); // false.
